src/main/java/Duke.java

```diff
@@ -1,10 +1,291 @@
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+
+/**
+ * Duke allows the user to maintain a list of tasks
+ */
 public class Duke {
+
```

Comment:
Hi, just a small inconsistency that I spotted. For some of your classes, there is an empty line between the class declaration and the class properties. However, some of the classes don't. 

Perhaps it will be more consistent if all classes have an empty line in between or none of them have an empty line in between?